<?xml version="1.0"?>


<launch>
	<node name="navigation_trajectory_follower" pkg="navigation_trajectory_follower" type="trajectory_follower_node" output="screen">
		<param name="cycleFrequencyInHz" type="double" value="50.0"/>

		<param name="positionGainTranslation" type="double" value="0.5"/>
        	<param name="velocityGainTranslation" type="double" value="1.0"/>

		<param name="positionGainRotation" type="double" value="0.1"/>
        	<param name="velocityGainRotation" type="double" value="1.0"/>

		<param name="inputTrajectoryTopic" type="string" value="localTrajectory"/>
		<param name="inputOdometryTopic" type="string" value="odom"/>
		<param name="outputVelocityTopic" type="string" value="cmd_vel"/>
	</node>

</launch>

